taxonomy-editor-frontend/src/pages/startproject/index.tsx

```diff
@@ -32,9 +45,11 @@ export const StartProject = () => {
 
   const findDefaultBranchName = useCallback(() => {
     if (taxonomyName === "" || ownerName === "") return "";
-    return `${toSnakeCase(taxonomyName.toLowerCase())}_${ownerName
-      .replace(" ", "")
-      .toLowerCase()}_${Math.floor(Date.now() / 1000)}`;
+    return `${taxonomyName.toLowerCase()}_${ownerName}_${formatDate(
```

Comment:
nit: the `toLowerCase` for the taxonomy name is not needed because you use it again on the whole string after